homeassistant/components/habitica/quality_scale.yaml

```diff
@@ -0,0 +1,80 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
```

Comment:
The dependency `habitipy` is not built in the CI

Reply:
True. And this means this rule isn't fullfillable until the habitipy dependency has been removed, that will be at least until 2025.6.0